src/views/v2/loader.php

```diff
@@ -0,0 +1,3 @@
+<div class="tribe-events-view-loader tribe-common-a11y-hidden">
```

Comment:
A note on this. It'd be nice to use `role="alert"` and `aria-live="assertive"` for accessibility here, but we need custom JS to interact with this (add Loading text, add Loaded text). Not high priority to add this, but something to consider down the road.